src/chart/bar/BarView.ts

```diff
@@ -1176,7 +1176,8 @@ function createLarge(
     el.barWidth = barWidth;
     group.add(el);
     el.useStyle(data.getVisual('style'));
-
+    // Reset stroke color
+    el.style.stroke = null;
```

Comment:
Setting stroke to be null is not acceptable because users can set `largeThreshold` to a very small number like 10 and large rendering will be used even if the bar-width much more wider than 1px. In this case, setting stroke to be null is not expected.

Reply:
`el.style.strokeFirst = true;` makes sure stroking is rendered before filling so that filling colors will not be overlapped by stoking. The result is as follows:

![image](https://github.com/user-attachments/assets/ef01461d-fc86-4a06-a0b3-9efc3068c494)

The filling color is blue and the stroking color is red in the above case.

If this is expected result, you may update your PR like that.

Here's the test option:

```js
var xAxisData = [];
var data1 = [];
for (var i = 0; i < 10000; i++) {
    xAxisData.push('类目' + i);
    data1.push((Math.random() * 5).toFixed(2));
}

var option = {
    legend: {
        data: ['bar'],
        align: 'left'
    },
    tooltip: {},
    dataZoom: [{
        start: 30,
        end: 100,
        type: 'slider'
    }],
    xAxis: {
        data: xAxisData,
        silent: false,
        splitLine: {
            show: false
        },
        splitArea: {
            show: false
        }
    },
    yAxis: {
        splitArea: {
            show: false
        }
    },
    series: [{
        name: 'bar',
        type: 'bar',
        large: true,
        largeThreshold: 10,
        data: data1,
        itemStyle: {
            borderWidth: 1,
            borderColor: 'red'
        }
    }]
}
```

Reply:
Thanks @Ovilia! I updated my code based on your review, and your solution resolved the issue. The only thing I noticed is that, in a larger chart there seems to be a darker area. I suspect this might be caused by overlapping bars, but I'm not really sure if it’s an actual problem. Have you encountered something like this in your environment?
(I use a 1920x1080 monitor and used the option that you  provided)

Have a great day!

![image](https://github.com/user-attachments/assets/9df8a28d-04ca-47d9-b35e-719777c96e65)

Reply:
The part inside yellow windows seems to be correct but the ones having redish shadow don't seem quite right. I will have a double check with other committers. Thanks for letting me know!

Reply:
I checked the SVG of rendered path, and it seemed to be all right. Maybe it's a problem of SVG/Canvas implementation. After discussing with others, we would suggest setting `stroke = null` as you did in the previous commit. Sorry for your time!

Reply:
> I checked the SVG of rendered path, and it seemed to be all right. Maybe it's a problem of SVG/Canvas implementation. After discussing with others, we would suggest setting `stroke = null` as you did in the previous commit. Sorry for your time!

Thank you, I will try to see if I can find a better solution in general. No worries =)

Reply:
I tried but I didn't find a better solution. I remodified the file to reinclude 
`el.style.stroke = null;` in place of `el.style.strokeFirst = true;`